adb_graphics/default_specs.yml

```diff
@@ -115,6 +115,111 @@ acpcp: # Accumulated run total precipitation
     ticks: 0
     transform: conversions.kgm2_to_in
     unit: in
+  ens1:
+    clevs: [0., 0.01, 0.1, 0.25, 0.5, 1, 2, 3, 4, 6, 10]
+    cmap: gist_ncar
+    colors: rainbow12_colors
+    ncl_name: APCP_P12_L1_{grid}_acc1h
+    ticks: 0
+    title: Ensemble Mean 1-hr Precipitation
+    transform: conversions.kgm2_to_in
+    unit: in
+  ens24:
+    clevs: [0., 0.01, 0.1, 0.25, 0.5, 1, 2, 3, 4, 6, 10]
+    cmap: gist_ncar
+    colors: rainbow12_colors
+    ncl_name: APCP_P12_L1_{grid}_acc24h
+    ticks: 0
+    title: Ensemble Mean 24-hr Precipitation
+    transform: conversions.kgm2_to_in
+    unit: in
+  ens6:
+    clevs: [0., 0.01, 0.1, 0.25, 0.5, 1, 2, 3, 4, 6, 10]
+    cmap: gist_ncar
+    colors: rainbow12_colors
+    ncl_name: APCP_P12_L1_{grid}_acc6h
+    ticks: 0
+    title: Ensemble Mean 6-hr Precipitation
+    transform: conversions.kgm2_to_in
+    unit: in
+acpcpens1: # ensemble probability of precipitation, 1h
+  76.2mm: &ens_precip_prob1
```

Comment:
I'm curious if we could get even more re-use of this anchor here? Could we just use `&ens_precip_prob` and then change only the ncl_name in the different accumulation bins below?

Reply:
made that change, and also added another anchor for ensemble means.